lib/api.js

```diff
@@ -20,6 +20,7 @@ function PlexAPI(options, deprecatedPort) {
     this.hostname = hostname;
     this.port = deprecatedPort || opts.port || PLEX_SERVER_PORT;
     this.https = opts.https;
+    this.timeout = opts.timeout || 60000;
```

Comment:
Adding a line in `Readme.md` to document this new **timeout** option would be great.